src/library.js

```diff
@@ -4581,32 +4581,26 @@ LibraryManager.library = {
     return STACK_BASE;
   },
 
+  _readAsmConstArgsArray: '=[]',
```

Comment:
Use $ prefix? (no underscores needed).

Reply:
Same rationale as in https://github.com/emscripten-core/emscripten/pull/11226#discussion_r429559110